extensions/wikia/ThemeDesigner/ThemeDesignerHooks.class.php

```diff
@@ -61,12 +55,14 @@ public static function onUploadComplete( $image ) {
 	 * @return bool return false to prevent the upload
 	 */
 	public static function onUploadVerification( $destName, $tempPath, &$error ) {
-		$destName = strtolower($destName);
+		$destName = strtolower( $destName );
 		if ( $destName == 'wiki-wordmark.png' || $destName == 'wiki-background' ) {
 			// BugId:983
 			$error = wfMessage( 'themedesigner-manual-upload-error' )->plain();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change the output format of this i18n string away from plain(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)